docs/resources/Guild.md

```diff
@@ -999,39 +1047,6 @@ Modify a [guild widget](#DOCS_RESOURCES_GUILD/guild-widget-object) object for th
 
 Returns the widget for the guild.
```

Comment:
Would be nice to link widget to the Get Guild Widget Object section above.

Reply:
Done 👍